homeassistant/components/sonos/quality_scale.yaml

```diff
@@ -0,0 +1,73 @@
+rules:
+  # Bronze
+  action-setup:
+    status: todo
+    comment: |
+      Needs complete refactor
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: todo
+  test-before-configure: done
+  test-before-setup:
+    status: exempt
+    comment: |
+      Offline devices are automatically reconnected
+  unique-config-entry:
+    status: exempt
+    comment: |
+      Only allows a single entry
```

Comment:
You can also set single_config_entry in the manifest

Reply:
ok. will try that. when I marked it as done the CI fails since it doesn't detect the right exception being raised.